lang/test/org/partiql/lang/errors/ParserErrorsTest.kt

```diff
@@ -1623,10 +1673,10 @@ class ParserErrorsTest : TestBase() {
     @Test
     fun execAtUnexpectedLocationInExpression() = checkInputThrowingParserException(
         "SELECT * FROM (EXEC undrop 'foo')",
-        ErrorCode.PARSE_EXEC_AT_UNEXPECTED_LOCATION,
+        ErrorCode.PARSE_UNEXPECTED_TERM,
         mapOf(
             Property.LINE_NUMBER to 1L,
-            Property.COLUMN_NUMBER to 16L,
-            Property.TOKEN_TYPE to TokenType.KEYWORD,
-            Property.TOKEN_VALUE to ion.newSymbol("exec")))
+            Property.COLUMN_NUMBER to 21L,
+            Property.TOKEN_TYPE to TokenType.IDENTIFIER,
+            Property.TOKEN_VALUE to ion.newSymbol("undrop")))
```

Comment:
Shouldn't the unexpected term be `EXEC` here?

Reply:
It should be. But the associated token with the `EXEC` ParseNode is the name of the stored procedure. Hence, the issue. Ti get around with this, I think we can pass the name of the stored procedure in the arguments along with the other `arglist` or some such. But that's worthy of a separate issue.